editor/editor_properties.cpp

```diff
@@ -936,6 +993,11 @@ class EditorPropertyLayersGrid : public Control {
 	}
 };
 
+void EditorPropertyLayers::set_read_only(bool p_read_only) {
+	button->set_disabled(p_read_only);
+	EditorProperty::set_read_only(p_read_only);
+};
```

Comment:
This is not enough to disable this control, there's still the grid of clickable buttons.